src/constants/themes/index.ts

```diff
@@ -1,3 +1,3 @@
 export { primaryTheme } from './primaryTheme';
 export { secondaryTheme } from './secondaryTheme';
-export { COLORS_PROP_TYPES } from './types';
+export * from './types';
```

Comment:
Rather than have in all our source files:

```tsx
import { SPACER } from '../../constants';
import { ThemeType } from '../../constants/theme/types';
```

We can just combine them for convenience: 

```tsx
import { SPACER, ThemeType } from '../../constants';
```